docs/config/index.md

```diff
@@ -82,6 +82,30 @@ export default defineConfig(async ({ command, mode }) => {
 })
 ```
 
+
+## Hot reload monorepos packages
+
+To have your library packages hot reload in local dev
+
+1. Ensure your workspace is set in your `apps/main-app/package.json`:
+
+```js
+"@package/example": "workspace:packages/example"
+```
+
+2. Update the exports property in the `packages/example/package.json` of `@package/example`:
```

Comment:
if you already have one do you need to update it in some way? if not, it should probably say "include" rather than "update"

why is this necessary?